introduction.md

```diff
@@ -0,0 +1 @@
+Hello, I am Meru, working as BA/PM in Germany-FFM and doing some coding in my sparetime.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.